temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // The namespace to route requests to. This field is mutable.
+    string namespace = 2;
+
+    // The task queue to route requests to. This field is mutable.
+    string task_queue = 3;
+
+    // The set of authorization policies for the service. Each request's caller
+    // must match with at least one of the specs to be accepted by the incoming service.
+    repeated IncomingServiceAuthorizationSpec authorization_specs = 4;
+}
+
+message IncomingServiceAuthorizationSpec {
+    // The policy set on this incoming service authorization spec.
+    // Possible values are: allowed_cloud_namespace
+    string policy_type = 1;
```

Comment:
Why not make this an enum, I'm curious?

Reply:
@anekkanti can comment here. We appear to have stayed away from enums elsewhere (and are using strings instead), so this is just being consistent with the existing cloud APIs